mainapp/views.py

```diff
@@ -267,6 +274,31 @@ def mapdata(request):
     cache.set("mapdata:" + district, data, settings.CACHE_TIMEOUT)
     return JsonResponse(list(data) , safe=False)
 
+@csrf_exempt
+def parsedata(request):
+    return_data = list()
+    data = json.loads(request.body)
```

Comment:
When I tried testing with Postman, I had to change this line to `request.body.decode('utf8')` to get it working. Else there was a type error - `the JSON object must be str, not 'bytes'`

Reply:
I tried with ``curl -i -X POST -H 'Content-Type: application/json' -d '{ "messages" ... }' <url>`` and it had worked out of the box.

Reply:
I tried with Postman and curl and it worked fine. Can you check once with curl command from your machine and get back once. Also, can you check the default request encoding that is enabled in your version of Postman?

Reply:
Tried curl and Postman both, and still no luck. But if it's been validated by you folks then forget about it.